lib/rules/no-irregular-whitespace.js

```diff
@@ -61,26 +102,13 @@ module.exports = function(context) {
              * When writing this code also evaluating per node was missing out connecting tokens in some cases
              * We can later filter the errors when they are found to be not an issue in nodes we don't care about
              */
-            var sourceLines = context.getSourceLines();
-
-            sourceLines.forEach(function (sourceLine, lineIndex) {
-                var location,
-                    match = irregularWhitespace.exec(sourceLine);
 
-                if (match !== null) {
-                    location = {
-                        line: lineIndex + 1,
-                        column: match.index
-                    };
-
-                    errors.push([node, location, "Irregular whitespace not allowed"]);
-                }
-            });
+            checkForIrregularWhitespace(node);
+            checkForIrregularLineTerminators(node);
         },
+
         "Identifier": removeInvalidNodeErrors,
         "Literal": removeInvalidNodeErrors,
-        "Statement": removeInvalidNodeErrors,
-        "Expression": removeInvalidNodeErrors,
```

Comment:
I removed `"Statement"` and `"Expression"` because I don’t think those node types even exist.

I’m not sure about `"Identifier"`, everything seems to work if I remove this listener.